build/configs/artik055s/audio/defconfig

```diff
@@ -1145,14 +1138,15 @@ CONFIG_EXAMPLES_SLSIWIFI_STACKSIZE=2048
 # CONFIG_EXAMPLES_ST_THINGS is not set
 CONFIG_EXAMPLES_TESTCASE=y
 # CONFIG_DISABLE_MANUAL_TESTCASE is not set
-# CONFIG_EXAMPLES_TESTCASE_ARASTORAGE_UTC is not set
-# CONFIG_EXAMPLES_TESTCASE_ARASTORAGE_ITC is not set
-CONFIG_EXAMPLES_TESTCASE_AUDIO_UTC=y
-CONFIG_EXAMPLES_TESTCASE_AUDIO_ITC=y
+# CONFIG_EXAMPLES_TESTCASE_DRIVERS is not set
 # CONFIG_EXAMPLES_TESTCASE_FILESYSTEM is not set
 # CONFIG_EXAMPLES_TESTCASE_KERNEL is not set
 # CONFIG_EXAMPLES_TESTCASE_NETWORK is not set
 # CONFIG_EXAMPLES_TESTCASE_TTRACE is not set
+# CONFIG_EXAMPLES_TESTCASE_ARASTORAGE_UTC is not set
```

Comment:
hmmm please apply only rammtd features.

Reply:
Modifying through make menuconfig is the correct way. when you run make menuconfig it might add new configs with not set. we cant avoid this. it is okay as long as it is not set.